Yandex.Cloud.SDK/Yandex.Cloud.SDK.csproj

```diff
@@ -18,7 +18,9 @@
     </PropertyGroup>
 
     <ItemGroup>
+        <PackageReference Include="BouncyCastle.NetCore" Version="1.8.10" />

```

Comment:
This package can be omitted when targeting net5.0 and higher, since [this method](https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.rsa.importfrompem?view=net-7.0) was introduced. If you still need to maintain compatibility with .NET Framework, multi-targeting is an option, so it will look like this:
```xml
    <PropertyGroup>
        <TargetFrameworks>netstandard2.0;net7.0</TargetFrameworks>
        <!-- omitted for brevity -->
     </PropertyGroup>

    <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
        <PackageReference Include="BouncyCastle.NetCore" Version="1.8.10" />
    </ItemGroup>
```

Also the official BouncyCastle package was [published on NuGet](https://www.nuget.org/packages/BouncyCastle.Cryptography) – however, it has a significantly larger size, which might be undesirable, so I can recommend to use [Portable.BouncyCastle](https://www.nuget.org/packages/Portable.BouncyCastle#usedby-body-tab) which is [officially recognised](https://www.bouncycastle.org/csharp/index.html#DOWNLOAD2000) by developers of BouncyCastle, instead of BouncyCastle.NetCore.